MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/basics/KotlinSimpleMapViewActivity.kt

```diff
@@ -25,7 +25,8 @@ class KotlinSimpleMapViewActivity : AppCompatActivity() {
         mapView?.getMapAsync { mapboxMap ->
             mapboxMap.setStyle(Style.MAPBOX_STREETS) {
 
-                // Map is set up and the style has loaded. Now you can add data or make other map adjustments.
+                // Map in fragment container is set up and the style has loaded.
+                // Now you can add data or make other map adjustments.
```

Comment:
Should we keep the original comment since this is the KotlinSimpleMapViewActivity and fragment container is not used in this example?